shared/common-adapters/header-hoc.native.js

```diff
@@ -51,7 +51,6 @@ const _headerStyle = {
   borderBottomColor: globalColors.black_05,
   borderBottomWidth: StyleSheet.hairlineWidth,
   justifyContent: 'flex-start',
-  marginTop: statusBarHeight,
```

Comment:
cc: @chromakode this is to fix the double marginTop. we can discuss if this doesn't make sense